.github/oxid-esales/defaults/component.yaml

```diff
@@ -3,7 +3,7 @@ install:
   cache:
     prepared_shop: false
   git:
-    repository: '{{ .Data.global.repo }}'
+    repository: '{{ .Data.install.git.repository }}'
```

Comment:
This line is not really necessary. In the defaults, we define install.git.repository and that is the default value. Now we overwrite the default value with the default value.